internal/stores/stores.go

```diff
@@ -33,6 +34,15 @@ type Store interface {
 	GetFsys() afero.Fs
 }
 
+var (
+	// ensure AzureblobStore meets Store interface
+	_ = Store(&AzureBlobStore{})
+	// ensure S3Store meets Store interface
+	_ = Store(&S3Store{})
+	// ensure GCSStore meets Store interface
+	_ = Store(&GCSStore{})
+)
+
```

Comment:
What is this for? I don't remember needing this for GCS.

Reply:
I don't think it's strictly necessary but it made it more obvious to me when I accidentally broke the interface implementations.

Reply:
How does this help? I'm 100% not following 😭